includes/classes/InternalConnections/NetworkSiteConnection.php

```diff
@@ -267,6 +267,12 @@ public function pull( $items ) {
 
 			restore_current_blog();
 
+			// Allow the sync'ed post to be updated via a REST request to
+			// get the rendered content.
+			if ( apply_filters( 'dt_pull_post_apply_rendered_content', true, $new_post_id, $this, $post_array ) ) {
```

Comment:
Not a CR comment but wondering if we want to default this to `false`? So sites that need the extra step of pulling in rendered content can opt-in but this won't impact other setups that don't need this? cc/ @jeffpaul

Reply:
I was thinking the same thing, with an eventual option in the settings to turn that off without a code change.

Reply:
My only concern is we're currently doing this for pushed posts so I imagine the perception is that we'd do this by default for pulled posts as well.  I _think_ I'd rather support the broader case and allow sites that know they don't need to render content first before pulling could adjust the default to `false` and gain back whatever performance the HTTP call is impacting.  Unless I'm misunderstanding things or not fully thinking through other scenarios, if so please call those out.